frontend/src/components/instructions/Instructions.js

```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react'
+import InstructionsDisplayer from "./InstructionsDisplayer";
+
+export default class Instructions extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            dropdownModule: false,
+            dropdownClass: false,
+        }
+    }
+
+//trigger the boolean in state with str as input
+//reusable function
+    triggerDropdown = (str) => () => {
+        this.setState({[str]: !this.state[str]})
+    };
+
+//Put input value in state with name of the input as name of the variable
```

Comment:
/**
* Put input value in state with name of the input as name of the variable
* @param {String} str 
* @returns void 
*/